src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/UpdateSubmission.java

```diff
@@ -28,6 +29,9 @@ public class UpdateSubmission {
     public static final String ADDITIONS_FIELD = "additions";
     public static final String CESSATIONS_FIELD = "cessations";
     public static final String TRUSTS_ADDITIONS_FIELD = "trusts";
+    public static final String CHANGE_BO_RELEVANT_PERIOD = "changeBORelevantPeriod";
+    public static final String TRUSTEE_INVOLVED_RELEVANT_PERIOD = "trusteeInvolvedRelevantPeriod";
```

Comment:
Just wondering if this constant and the associated field should also be prefixed with 'change'?

Reply:
checked it should be left like this